src/app/helpers/file-size.utils.ts

```diff
@@ -53,8 +53,8 @@ function normalizeFileSizeBase10(value: number, baseUnit: 'b' | 'B'): [formatted
   }
   formatted = Math.round((formatted + Number.EPSILON) * 100) / 100;
   switch (increment) {
-    case kb:
-      return [formatted, 'k' + baseUnit];
+    case Kb:
```

Comment:
This is incorrect.
https://en.wikipedia.org/wiki/Data-rate_units#:~:text=Kilobit%20per%20second%20(symbol%20kbit,1%2C000%20bits%20per%20second